lib/floki/selector/pseudo_class.ex

```diff
@@ -25,6 +25,17 @@ defmodule Floki.Selector.PseudoClass do
     false
   end
 
+  def match_contains?(tree, html_node, %__MODULE__{value: value}) do
+    res = Enum.filter(html_node.children_nodes_ids, fn(id) ->
```

Comment:
I added a test for this as well. 

> String.split(content, " ") |> Enum.any?(& &1 == value)

I think substring match (`=~`) might be a little better, so you can do things like `fl-contains('violets are'),